pkgs/applications/gis/grass/default.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation {
   readline ffmpeg makeWrapper wxGTK30 netcdf geos postgresql mysql.connector-c blas ]
     ++ (with python2Packages; [ python dateutil wxPython30 numpy ]);
 
+  # On Darwin the installer tries to symlink the help files into a system
+  # directory
+  patches = [ ./noLn.patch ];
```

Comment:
make this optional for darwin?

Reply:
Also, two lowercase letters followed by an uppercase letter and again a lowercase letter? I'm sure that's not necessary. Maybe `no_symbolic_links.patch`?